src/c/dune

```diff
@@ -122,10 +122,17 @@ let () = Jbuild_plugin.V1.send @@ {|
  (targets generate_types_step_2.exe)
  (deps (:c generate_types_step_2.c) helpers.h shims.h)
  (action (bash "\
-  %{cc} %{c} \
-  -I '%{lib:ctypes:.}' \
-  -I %{ocaml_where} \
-  |}^ i_option ^{| -o %{targets}")))
+  if [ '%{ocaml-config:ccomp_type}' = 'msvc' ]; then \
+    %{cc} %{c} \
+    -I '%{lib:ctypes:.}' \
+    -I %{ocaml_where} \
+    |}^ i_option ^{| /Fe\"%{targets}\"; \
+  else \
+    %{cc} %{c} \
+    -I '%{lib:ctypes:.}' \
+    -I %{ocaml_where} \
+    |}^ i_option ^{| -o %{targets}; \
+  fi")))
```

Comment:
As far as I can tell at first glance, only this change and the change to `src/c/windows_version.h` are necessary for Luv itself. Is that right?

Reply:
Correct. The rest is regression test apparatus for Windows. If you want, I can split the PR when I get back in next week.

Reply:
Yes, please. In the long term (not in this PR), the test suite should be made compatible by porting more of it to Windows, and then it can be run directly from the CI without intervening scripts. So I prefer not to merge the scripts, but keep them in a branch for reference.

Reply:
1. Branch with test scripts is [https://github.com/jonahbeckford/luv/tree/0.5.11-msvctesting](https://github.com/jonahbeckford/luv/tree/0.5.11-msvctesting)
2. Did a force push that removes test scripts.